modules/linear_boltzmann_solvers/lbs_solver/iterative_methods/lbs_iterative_methods.h

```diff
@@ -12,8 +12,8 @@ namespace lbs
 enum class IterativeMethod : int
 {
   NONE = 0,
-  CLASSICRICHARDSON = 1,        ///< Otherwise known as Source Iteration
-  CLASSICRICHARDSON_CYCLES = 2, ///< Source Iteration with Cycles support
+  // CLASSICRICHARDSON = 1,        ///< Otherwise known as Source Iteration
+  // CLASSICRICHARDSON_CYCLES = 2, ///< Source Iteration with Cycles support
```

Comment:
Are there any plans on recycling those 2 values in future? If not, then delete those lines...

Reply:
Yes, classic Richardson will make a comeback as soon as possible. I believe it's even slated for 1.0.

Reply:
Ok.